### ISSUE TITLE
Backport attribution check

### ISSUE THREAD

**Issue by rinOfTheStars** [Enchanted-Games/no-elytra-boost#1]
As I'm currently thinking of making a modpack for 1.20.1 that'd make good use of this mod, I went and backported it to said version of minecraft + lexforge [here](https://github.com/cocona20xx/no-rocket-boosting-backport). Have I correctly attributed everything for the backport version of the mod? Not super familiar with how CC BY-NC 4.0 works when it comes to software, hence my question. Also, if I publish this backport to modrinth, would you like to be included on the project as an author (since most of the work was yours)?

**Comment by Enchanted-Games** [comment]
Yep that looks good, the main thing with the license is that you can’t sell my work without making significant changes.
You can post it on modrinth and add me as an author that’s fine with me. Or you can open a pull request and I can post the backport on my modrinth page, whichever you’d prefer
